snap/snapcraft.yaml

```diff
@@ -44,37 +44,18 @@ parts:
     source-type: git
     source-tag: xena-last
     python-packages:
-    - confluent-kafka==1.8.2
     - git+https://opendev.org/openstack/barbican-tempest-plugin.git@xena-last
-    - git+https://opendev.org/openstack/blazar-tempest-plugin.git@xena-last
     - git+https://opendev.org/openstack/cinder-tempest-plugin.git@xena-last
-    - git+https://opendev.org/openstack/cloudkitty-tempest-plugin.git@xena-last
-    - git+https://opendev.org/openstack/cyborg-tempest-plugin.git@2.2.0
     - git+https://opendev.org/openstack/designate-tempest-plugin.git@xena-last
-    - git+https://opendev.org/openstack/ec2api-tempest-plugin.git@xena-last
-    - git+https://opendev.org/openstack/freezer-tempest-plugin.git@xena-last
-    - git+https://opendev.org/openstack/glance-tempest-plugin.git@0.7.0
+    - git+https://opendev.org/openstack/glance-tempest-plugin.git@0.8.0
```

Comment:
by the release repo I think this should be [0.2.0](https://opendev.org/openstack/releases/src/branch/master/deliverables/xena/glance-tempest-plugin.yaml#L12). Can you confirm it?

Reply:
right, but since there's no xena-last tag, we should grab the latest version available. See https://opendev.org/openstack/glance-tempest-plugin